src/systems/material.js

```diff
@@ -195,6 +205,12 @@ module.exports.System = registerSystem('material', {
   }
 });
 
+function isHLS (videoEl) {
+  if (videoEl.type === 'application/x-mpegurl') return true;
```

Comment:
i've only seen lowercase, but good question.

not sure on your other two points